src/containers/Aside/index.tsx

```diff
@@ -0,0 +1,53 @@
+import CNavLink from 'src/components/CNavLink';
+
+import { navLinks } from 'src/constants/navlinks';
+import { Pages } from 'src/constants/pages';
+
+import GearSix from 'src/svgs/GearSix';
+import LifeBuoy from 'src/svgs/LifeBuoy';
+import SquareHalf from 'src/svgs/SquareHalf';
+
+type AsideProps = {
+  isMinimized: boolean;
+  onClick: () => void;
```

Comment:
`onClick` is not an specific word. Use specific words like `onMinimized` or something relevant.